api/v1alpha1/containerized_types.go

```diff
@@ -28,7 +28,7 @@ type PodSpecWorkloadSpec struct {
 	// Replicas is the desired number of replicas of the given podSpec.
 	// These are replicas in the sense that they are instantiations of the same podSpec.
 	// If unspecified, defaults to 1.
-	Replicas *int32 `json:"replicas"`
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
@ryanzhang-oss I set this field as optional to fix the issue https://github.com/oam-dev/kubevela/pull/355#discussion_r502791069. Now I won't set Controller to true when setting the workload's `ownerRef` to trait. In this way, the replicas of workload won't be refreshed to the original `spec.replicas` by workload when the filed is not set in manifest. For the sample, please refer to folder `samples/autoscalers/PodSpecWorkload`. Please review it, thanks.